packages/flutter/lib/src/foundation/annotations.dart

```diff
@@ -33,21 +32,53 @@
 ///
 /// See also:
 ///
-///  * [DocumentationIcon], which is used to give the URL to an image that
-///    represents the class.
+///  * [Subcategory], to help place the class in a subcategory index.
 ///  * [Summary], which is used to provide a one-line description of a
 ///    class that overrides the inline documentations' own description.
 class Category {
   /// Create an annotation to provide a categorization of a class.
-  const Category(this.sections) : assert(sections != null);
+  const Category(this.value) : assert(value != null);
 
-  /// The strings the correspond to the section and subsection of the
-  /// category represented by this object.
+  /// The string that corresponds to the category represented by this object.
   ///
-  /// By convention, this list usually has two items. The allowed values
-  /// are defined by the team that owns the library to which the annotated
-  /// class belongs.
-  final List<String> sections;
+  /// The allowed values are defined by the team that owns the library to which
+  /// the annotated class belongs.
+  final String value;
+}
+
+/// A subcategory with which to annotate a class, for documentation purposes.
+///
+/// The engineering team that owns the library to which the class belongs
+/// defines the categories and subcategories used for classes in that library.
+///
+/// A class can have multiple subcategories.
+///
+/// {@tool sample}
+///
+/// ```dart
+/// /// ...documentation...
+/// @Category('Pots')
+/// @Subcategory('Coffee')
+/// class FrenchPress {
+///   // ...code...
+/// }
+/// ```
+/// {@end-tool}
+///
+/// See also:
+///
+///  * [Subcategory], to help place the class in a subcategory index.
```

Comment:
This should presumably link back to Category?

Reply:
Yup, I'll update.